src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java

```diff
@@ -35,7 +40,7 @@ public SendEmailNotificationResponse execute(UUID userId, UUID templateId) {
         }
 
         EmailNotification emailNotification = new EmailNotification(
-                userEmailAddress.get(),
+                userEmailAddress.get().getEmail(),
```

Comment:
in case optional is empty it will generate an exception

Reply:
There is a check for userEmailAdress, in line 38, I don't know why is hidden. 🔍